gulpfile.js

```diff
@@ -0,0 +1,57 @@
+var gulp = require("gulp");
+
+var mochaPhantomJS = require('gulp-mocha-phantomjs');
+gulp.task("test", ["build-debug"], function() {
+  return gulp.src("tests/SpecRunner.html")
+    .pipe(mochaPhantomJS());
+});
+
+var webserver = require("gulp-webserver");
+gulp.task("test-browser", ["build-debug"], function() {
+  gulp.src(".")
+    .pipe(webserver({
+      livereload: true,
+      open: "tests/SpecRunner.html"
+    }));
+});
+
```

Comment:
Both `test` and `test-browser` depend on `build-debug` because they use `dist/amd-resolver.js` - which is created by running `build-debug`.